components/server/src/prometheus-metrics.ts

```diff
@@ -179,6 +179,7 @@ export type FailedInstanceStartReason =
     | "startOnClusterFailed"
     | "imageBuildFailed"
     | "resourceExhausted"
+    | "clusterMaintenance"
```

Comment:
Would it make sense to make this `workspaceClusterMaintenance` to be explicit that it's not webapp maintenance?